src/DTOs/PdfPage.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Spatie\PdfToImage\DTOs;
+
+use Spatie\PdfToImage\Enums\OutputFormat;
+
+class PdfPage
+{
+    public readonly int $number;
+
+    public readonly OutputFormat $format;
+
+    public readonly string $prefix;
+
+    public readonly string $path;
+
+    public function __construct(int $pageNumber, OutputFormat $format, string $prefix, string $path)
+    {
+        $this->number = $pageNumber;
+        $this->format = $format;
+        $this->prefix = $prefix;
+        $this->path = $path;
+    }
+
+    public static function make(int $pageNumber, OutputFormat $format, string $prefix, string $path): self
+    {
+        return new self($pageNumber, $format, $prefix, $path);
+    }
+
+    public function getFilename(): string
```

Comment:
What about dropping the get prefix, and just using fileName() ?

Reply:
That's a great suggestion and in line with the other method names, thanks! :+1: